cocos/ui/scroll-view.ts

```diff
@@ -1240,13 +1276,12 @@ export class ScrollView extends ViewGroup {
 
         const originalMoveLength = deltaMove.length();
         let factor = targetDelta.length() / originalMoveLength;
-        targetDelta.add(deltaMove);
```

Comment:
as targetDelta will be set in `if (this.brake > 0 && factor > 7)`, so move it to `else` block bellow.